planet_crs_registry/core/business/crs_response.py

```diff
@@ -49,54 +49,39 @@ class GmlResponse(Response):
 
     media_type = "application/xml"
 
-    def render(self, content: str) -> bytes:
-        """Renders the GML response.
-        The GML response is renders in two different ways:
-        - in prod : use gdalsrsinfo because the web service is installed
-        in the same image as gdal
-        - in dev : use gdal image
+    @staticmethod
```

Comment:
Je pense qu'il vaut mieux éviter d'en faire une fonction statique. [La doc de FastAPI montre un exemple de méthode render.(self)](https://fastapi.tiangolo.com/advanced/custom-response/#custom-response-class).

En fait, on devrait faire une méthode `render(self) -> bytes:` qui utilise `self.content` au lieu d'un contenu arbitraire.

Reply:
Solved in pull request #3